mri/tests/test_reconstructors.py

```diff
@@ -313,6 +313,54 @@ def test_check_asserts(self):
             num_iterations=self.num_iter,
         )
 
+    def test_stack3d_self_calibration_recon(self):
+        # This test carries out a self calibration recon using Stack3D
+        image = get_sample_data('3d-pmri')
+        # Restrict z data
+        z_loc = 5
+        image = image.data[
```

Comment:
You can use:
```python
image = image.data[..., image.shape[3]//2-z_loc):image.shape[3]//2+z_loc)]
```

Reply:
//2 returns int is it? Thats cool! fixing

Reply:
yes you can test that

Reply:
and you didn't use the ellipse `...` instead of `:, :, :,`

Reply:
Oh, thats also doable? I thought that it was just <whatever you did etc> stuff. 
Doing this change

Reply:
Done.

Reply:
yes I understand where the confusion comes from.